packages/ic-management/README.md

```diff
@@ -0,0 +1,59 @@
+# ckbtc-js
+
+A library for interfacing with [Internet Computer (IC) management canister](https://internetcomputer.org/docs/current/developer-docs/integrations/https-outcalls/https-outcalls-how-to-use/#ic-management-canister).
+
+[![npm version](https://img.shields.io/npm/v/@dfinity/management.svg?logo=npm)](https://www.npmjs.com/package/@dfinity/management) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+## Table of contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Features](#features)
+
+## Installation
+
+You can use `ic-management-js` by installing it in your project.
+
+```bash
+npm i @dfinity/ic-management
+```
+
+The bundle needs peer dependencies, be sure that following resources are available in your project as well.
+
+```bash
+npm i @dfinity/agent @dfinity/candid @dfinity/principal @dfinity/utils
+```
+
+## Usage
+
+The features are available through the class `LedgerCanister`. It has to be instantiated with a canister ID.
+
+e.g. fetching a token metadata.
+
+```ts
+import { ICMgmtCanister } from "@dfinity/ic-management";
+import { createAgent } from "@dfinity/utils";
+
+const agent = await createAgent({
+  identity,
+  host: HOST,
+});
+
+const { getCanisterDetails } = CkBTCCanister.create({
```

Comment:
This example uses `CkBTCCanister`, hehe.

Reply:
😅